package.json

```diff
@@ -4,8 +4,19 @@
   "description": "A tool to archive slack messages.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "ts-node src/index.ts",
```

Comment:
can we have this as `start:dev`, and have the `start` calling the built js file?